src/main/java/frc/robot/Robot.java

```diff
@@ -33,6 +37,11 @@ public class Robot extends TimedRobot {
 
     private RobotContainer robotContainer;
     private Command autoCommand;
+    private static final AtomicReference<RobotMode> mode = new AtomicReference<>(RobotMode.DISABLED);
+
+    public static RobotMode getMode(){
```

Comment:
a lot of people were reporting that it takes a lot of time and can cause CommandLoopOverruns because it has to request the information from the driver station itself